tests/entities_item_tests.py

```diff
@@ -625,4 +626,79 @@ def test_should_get_activity_logs(get_items, get_boards):
     # Assert
     ok_(activity_logs)
     eq_(activity_logs[0].id, id)
-    eq_(activity_logs[0].account_id, account_id)
\ No newline at end of file
+    eq_(activity_logs[0].account_id, account_id)
+
+@patch.object(en.Item,'get_column_values')
+@patch('moncli.api_v2.get_items') 
+def test_should_succeed_when_to_native_returns_a_str_when_passing_in_a_textvalue_value_with_api_get_itemdata(get_items,get_column_values):
```

Comment:
All of these tests need to be in **monday_types_tests.py** as specified in the docs.